examples/job-webhook-enhancer/handler.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+)
+
+type databricksWebhookContent struct {
+	EventType   string `json:"event_type"`
+	WorkspaceID int64  `json:"workspace_id"`
+	Run         struct {
+		RunID int64 `json:"run_id"`
+	}
+	Job struct {
+		JobID   int64  `json:"job_id"`
+		JobName string `json:"name"`
+	}
+}
+
+func databricksWebhookHandler(w http.ResponseWriter, r *http.Request) {
+	var jobMetadata jobs.Run
+	var err error = nil
+	JobWebhook := webhookParser(r)
+	switch EventType := JobWebhook.EventType; EventType {
```

Comment:
```suggestion
	switch JobWebhook.EventType {
```

you're not using `EventType` ;)